server/services/operations_service.go

```diff
@@ -125,18 +128,70 @@ func searchFilterFunc(query string) func(b *pb.Blurb) bool {
 	}
 }
 
+// returns a successful response if the resource name is not blank
 func (s operationsServerImpl) CancelOperation(ctx context.Context, in *lropb.CancelOperationRequest) (*empty.Empty, error) {
-	return nil, status.Error(codes.Unimplemented, "google.longrunning.CancelOperation is unimplemented.")
+	if in.Name == "" {
+		return nil, status.Error(codes.NotFound, "cannot cancel operation without a name.")
+	}
+	return &empty.Empty{}, nil
 }
 
+// returns a fixed response matching the given PageSize if the resource name is not blank
```

Comment:
Same thing regarding godoc comments needing to start with the method name.

Reply:
done